packages/auth-passport/lib/Passport.js

```diff
@@ -453,8 +479,7 @@ class PassportAuthStrategy {
 
     // Here we create both the Passport Session Item and the User Item
     // in KS5 as a single, nested mutation.
-    const sessionItem = await request(
```

Comment:
nice, this is something I did not like in the first place, glad to see it now removed